lib/registrant.rb

```diff
@@ -0,0 +1,53 @@
+class Registrant
+  attr_reader :age, :name, :permit, :license_data
```

Comment:
Careful here. We did not want to expose a method called `.permit`, we have the method `.permit?` to read us the value of the attribute, but we don't want to expose methods we don't actually need.